### PR TITLE
Fix the env and service names for AppSec throughput tests and benchmarks

### PR DESCRIPTION
**Pull request by andrewlock**
## Summary of changes

Fix the env and service names for AppSec throughput tests and benchmarks

## Reason for change

CI Visibility groups by env and service name, so we want these to all be included in the same run

## Implementation details

Add/change the required values

## Test coverage

Will see what the run looks like

## Other details
We may need to update the release dashaboards after merging to support these changes

### CODE REVIEW

File: .azure-pipelines/ultimate-pipeline.yml [hunk 1]
```diff
@@ -2700,7 +2700,8 @@ stages:
         ./run-appsec.sh "linux"
       displayName: Crank
       env:
-        DD_SERVICE: dd-trace-dotnet-appsec
+        DD_SERVICE: dd-trace-dotnet
```
Comment (pierotibou) [inline .azure-pipelines/ultimate-pipeline.yml:2703]: I assume the previous name wasn't used anywhere, could be worth a check with @robertpi

**Review by pierotibou — APPROVED** [review]

**Comment by andrewlock** [comment]
## Benchmarks Report :snail:

Benchmarks for #3413 compared to [master](https://github.com/DataDog/dd-trace-dotnet/tree/ffef158d4cd74d286f1cade3a2368289f21661c7): 
* All benchmarks have **the same** speed
* All benchmarks have **the same** allocations


The following thresholds were used for comparing the benchmark speeds:
* Mann–Whitney U test with statistical test for significance of **5%**
* Only results indicating a difference greater than **10%** and **0.3 ns** are considered.

Allocation changes below **0.5%** are ignored.


### Benchmark details


<details><summary><strong>Benchmarks.Trace.AgentWriterBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/ffef158d4cd74d286f1cade3a2368289f21661c7)|`WriteAndFlushEnrichedTraces`|net472|798μs|780ns|2.92μs|0.398| 0|0|3.22&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/ffef158d4cd74d286f1cade3a2368289f21661c7)|`WriteAndFlushEnrichedTraces`|netcoreapp3.1|539μs|466ns|1.81μs|0| 0|0|2.63&nbsp;KB|
|#3413|`WriteAndFlushEnrichedTraces`|net472|821μs|1.91μs|7.4μs|0.408| 0|0|3.22&nbsp;KB|
|#3413|`WriteAndFlushEnrichedTraces`|netcoreapp3.1|541μs|627ns|2.43μs|0| 0|0|2.63&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.AppSecBodyBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/ffef158d4cd74d286f1cade3a2368289f21661c7)|`AllCycleSimpleBody`|net472|182ns|0.0812ns|0.304ns|0.0701| 9.1E-05|0|441&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/ffef158d4cd74d286f1cade3a2368289f21661c7)|`AllCycleSimpleBody`|netcoreapp3.1|257ns|0.153ns|0.571ns|0.00595| 0|0|440&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/ffef158d4cd74d286f1cade3a2368289f21661c7)|`AllCycleMoreComplexBody`|net472|180ns|0.0508ns|0.197ns|0.0663| 0|0|417&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/ffef158d4cd74d286f1cade3a2368289f21661c7)|`AllCycleMoreComplexBody`|netcoreapp3.1|239ns|0.188ns|0.729ns|0.00563| 0|0|416&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/ffef158d4cd74d286f1cade3a2368289f21661c7)|`BodyExtractorSimpleBody`|net472|307ns|0.333ns|1.29ns|0.0574| 0|0|361&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/ffef158d4cd74d286f1cade3a2368289f21661c7)|`BodyExtractorSimpleBody`|netcoreapp3.1|239ns|0.114ns|0.441ns|0.00361| 0|0|272&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/ffef158d4cd74d286f1cade3a2368289f21661c7)|`BodyExtractorMoreComplexBody`|net472|16μs|4.13ns|14.9ns|1.21| 0.016|0|7.62&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/ffef158d4cd74d286f1cade3a2368289f21661c7)|`BodyExtractorMoreComplexBody`|netcoreapp3.1|12.8μs|5.92ns|22.2ns|0.0893| 0|0|6.75&nbsp;KB|
|#3413|`AllCycleSimpleBody`|net472|182ns|0.0814ns|0.305ns|0.0701| 9.18E-05|0|441&nbsp;B|
|#3413|`AllCycleSimpleBody`|netcoreapp3.1|241ns|0.128ns|0.497ns|0.00604| 0|0|440&nbsp;B|
|#3413|`AllCycleMoreComplexBody`|net472|191ns|0.0732ns|0.264ns|0.0662| 0|0|417&nbsp;B|
|#3413|`AllCycleMoreComplexBody`|netcoreapp3.1|238ns|0.153ns|0.573ns|0.00564| 0|0|416&nbsp;B|
|#3413|`BodyExtractorSimpleBody`|net472|285ns|0.489ns|1.89ns|0.0573| 0|0|361&nbsp;B|
|#3413|`BodyExtractorSimpleBody`|netcoreapp3.1|238ns|0.0943ns|0.34ns|0.00371| 0|0|272&nbsp;B|
|#3413|`BodyExtractorMoreComplexBody`|net472|16.2μs|16.9ns|65.4ns|1.21| 0.0161|0|7.62&nbsp;KB|
|#3413|`BodyExtractorMoreComplexBody`|netcoreapp3.1|12.6μs|3.23ns|12.1ns|0.088| 0|0|6.75&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.AspNetCoreBenchmark</strong> - Unknown :shrug: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/ffef158d4cd74d286f1cade3a2368289f21661c7)|`SendRequest`|net472|0ns|0ns|0ns|0| 0|0|0&nbsp;b|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/ffef158d4cd74d286f1cade3a2368289f21661c7)|`SendRequest`|netcoreapp3.1|176μs|242ns|939ns|0.262| 0|0|20.44&nbsp;KB|
|#3413|`SendRequest`|net472|0ns|0ns|0ns|0| 0|0|0&nbsp;b|
|#3413|`SendRequest`|netcoreapp3.1|176μs|202ns|781ns|0.264| 0|0|20.44&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.DbCommandBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/ffef158d4cd74d286f1cade3a2368289f21661c7)|`ExecuteNonQuery`|net472|1.84μs|1.27ns|4.91ns|0.158| 0.000918|0|995&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/ffef158d4cd74d286f1cade3a2368289f21661c7)|`ExecuteNonQuery`|netcoreapp3.1|1.36μs|0.701ns|2.72ns|0.0135| 0|0|984&nbsp;B|
|#3413|`ExecuteNonQuery`|net472|1.75μs|0.765ns|2.86ns|0.158| 0.000871|0|995&nbsp;B|
|#3413|`ExecuteNonQuery`|netcoreapp3.1|1.44μs|0.314ns|1.17ns|0.0136| 0|0|984&nbsp;B|
</details>


<details><summary><strong>Benchmarks.Trace.ElasticsearchBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/ffef158d4cd74d286f1cade3a2368289f21661c7)|`CallElasticsearch`|net472|2.43μs|0.868ns|3.36ns|0.19| 0|0|1.2&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/ffef158d4cd74d286f1cade3a2368289f21661c7)|`CallElasticsearch`|netcoreapp3.1|1.52μs|0.481ns|1.73ns|0.0152| 0|0|1.14&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/ffef158d4cd74d286f1cade3a2368289f21661c7)|`CallElasticsearchAsync`|net472|2.6μs|1.51ns|5.83ns|0.213| 0|0|1.34&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/ffef158d4cd74d286f1cade3a2368289f21661c7)|`CallElasticsearchAsync`|netcoreapp3.1|1.58μs|0.621ns|2.4ns|0.0174| 0|0|1.26&nbsp;KB|
|#3413|`CallElasticsearch`|net472|2.64μs|0.704ns|2.73ns|0.191| 0|0|1.2&nbsp;KB|
|#3413|`CallElasticsearch`|netcoreapp3.1|1.57μs|0.733ns|2.84ns|0.0156| 0|0|1.14&nbsp;KB|
|#3413|`CallElasticsearchAsync`|net472|2.52μs|2.69ns|10.4ns|0.213| 0|0|1.34&nbsp;KB|
|#3413|`CallElasticsearchAsync`|netcoreapp3.1|1.62μs|3.66ns|13.7ns|0.017| 0|0|1.26&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.GraphQLBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/ffef158d4cd74d286f1cade3a2368289f21661c7)|`ExecuteAsync`|net472|2.78μs|1.27ns|4.74ns|0.233| 0|0|1.47&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/ffef158d4cd74d286f1cade3a2368289f21661c7)|`ExecuteAsync`|netcoreapp3.1|1.72μs|0.56ns|2.1ns|0.0189| 0|0|1.39&nbsp;KB|
|#3413|`ExecuteAsync`|net472|2.71μs|2.53ns|9.81ns|0.233| 0|0|1.47&nbsp;KB|
|#3413|`ExecuteAsync`|netcoreapp3.1|1.71μs|0.566ns|1.96ns|0.019| 0|0|1.39&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.HttpClientBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/ffef158d4cd74d286f1cade3a2368289f21661c7)|`SendAsync`|net472|5.83μs|3.39ns|13.1ns|0.445| 0|0|2.82&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/ffef158d4cd74d286f1cade3a2368289f21661c7)|`SendAsync`|netcoreapp3.1|3.59μs|4.19ns|15.1ns|0.0357| 0|0|2.65&nbsp;KB|
|#3413|`SendAsync`|net472|5.83μs|2.15ns|7.76ns|0.445| 0|0|2.82&nbsp;KB|
|#3413|`SendAsync`|netcoreapp3.1|3.67μs|4.01ns|15ns|0.035| 0|0|2.65&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.ILoggerBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/ffef158d4cd74d286f1cade3a2368289f21661c7)|`EnrichedLog`|net472|3.12μs|1.69ns|6.1ns|0.298| 0|0|1.88&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/ffef158d4cd74d286f1cade3a2368289f21661c7)|`EnrichedLog`|netcoreapp3.1|2.48μs|1.16ns|4.36ns|0.0259| 0|0|1.91&nbsp;KB|
|#3413|`EnrichedLog`|net472|3.11μs|2.79ns|10.8ns|0.298| 0|0|1.88&nbsp;KB|
|#3413|`EnrichedLog`|netcoreapp3.1|2.59μs|0.782ns|2.82ns|0.0257| 0|0|1.91&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.Log4netBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/ffef158d4cd74d286f1cade3a2368289f21661c7)|`EnrichedLog`|net472|152μs|206ns|799ns|0.683| 0.228|0|4.72&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/ffef158d4cd74d286f1cade3a2368289f21661c7)|`EnrichedLog`|netcoreapp3.1|119μs|159ns|615ns|0| 0|0|4.55&nbsp;KB|
|#3413|`EnrichedLog`|net472|151μs|190ns|736ns|0.681| 0.227|0|4.72&nbsp;KB|
|#3413|`EnrichedLog`|netcoreapp3.1|120μs|112ns|433ns|0.0598| 0|0|4.55&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.NLogBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/ffef158d4cd74d286f1cade3a2368289f21661c7)|`EnrichedLog`|net472|5.65μs|1.66ns|6.45ns|0.579| 0.00281|0|3.65&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/ffef158d4cd74d286f1cade3a2368289f21661c7)|`EnrichedLog`|netcoreapp3.1|4.37μs|1.69ns|6.32ns|0.0547| 0|0|3.98&nbsp;KB|
|#3413|`EnrichedLog`|net472|5.62μs|2.65ns|10.3ns|0.578| 0.00282|0|3.65&nbsp;KB|
|#3413|`EnrichedLog`|netcoreapp3.1|4.46μs|1.56ns|5.84ns|0.0535| 0|0|3.98&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.RedisBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/ffef158d4cd74d286f1cade3a2368289f21661c7)|`SendReceive`|net472|2.18μs|1.05ns|4.06ns|0.226| 0|0|1.42&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/ffef158d4cd74d286f1cade3a2368289f21661c7)|`SendReceive`|netcoreapp3.1|1.75μs|0.575ns|2.07ns|0.0184| 0|0|1.37&nbsp;KB|
|#3413|`SendReceive`|net472|2.31μs|3.17ns|12.3ns|0.226| 0|0|1.42&nbsp;KB|
|#3413|`SendReceive`|netcoreapp3.1|1.76μs|7.64ns|29.6ns|0.0189| 0|0|1.37&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.SerilogBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/ffef158d4cd74d286f1cade3a2368289f21661c7)|`EnrichedLog`|net472|5.02μs|2.3ns|8.6ns|0.364| 0|0|2.3&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/ffef158d4cd74d286f1cade3a2368289f21661c7)|`EnrichedLog`|netcoreapp3.1|4.13μs|2.38ns|9.24ns|0.0248| 0|0|1.86&nbsp;KB|
|#3413|`EnrichedLog`|net472|5.13μs|2.43ns|8.76ns|0.364| 0|0|2.3&nbsp;KB|
|#3413|`EnrichedLog`|netcoreapp3.1|4.18μs|2.22ns|8.31ns|0.0251| 0|0|1.86&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.SpanBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/ffef158d4cd74d286f1cade3a2368289f21661c7)|`StartFinishSpan`|net472|1.15μs|0.217ns|0.814ns|0.136| 0|0|859&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/ffef158d4cd74d286f1cade3a2368289f21661c7)|`StartFinishSpan`|netcoreapp3.1|906ns|0.402ns|1.5ns|0.0108| 0|0|808&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/ffef158d4cd74d286f1cade3a2368289f21661c7)|`StartFinishScope`|net472|1.4μs|0.57ns|2.21ns|0.149| 0|0|939&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/ffef158d4cd74d286f1cade3a2368289f21661c7)|`StartFinishScope`|netcoreapp3.1|1.09μs|0.381ns|1.42ns|0.0126| 0|0|928&nbsp;B|
|#3413|`StartFinishSpan`|net472|1.15μs|0.264ns|0.989ns|0.136| 0|0|859&nbsp;B|
|#3413|`StartFinishSpan`|netcoreapp3.1|1.02μs|2.45ns|9.5ns|0.0111| 0|0|808&nbsp;B|
|#3413|`StartFinishScope`|net472|1.37μs|0.612ns|2.37ns|0.149| 0|0|939&nbsp;B|
|#3413|`StartFinishScope`|netcoreapp3.1|1.12μs|0.866ns|3.12ns|0.0122| 0|0|928&nbsp;B|
</details>


<details><summary><strong>Benchmarks.Trace.TraceAnnotationsBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/ffef158d4cd74d286f1cade3a2368289f21661c7)|`RunOnMethodBegin`|net472|1.52μs|0.378ns|1.46ns|0.148| 0|0|939&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/ffef158d4cd74d286f1cade3a2368289f21661c7)|`RunOnMethodBegin`|netcoreapp3.1|1.09μs|0.379ns|1.37ns|0.0125| 0|0|928&nbsp;B|
|#3413|`RunOnMethodBegin`|net472|1.43μs|0.803ns|2.89ns|0.149| 0|0|939&nbsp;B|
|#3413|`RunOnMethodBegin`|netcoreapp3.1|1.17μs|0.231ns|0.832ns|0.0123| 0|0|928&nbsp;B|
</details>

**Review by zacharycmontoya — APPROVED** [review]

**Comment by andrewlock** [comment]
## Code Coverage Report :bar_chart:

:warning: Merging #3413 into [master](https://github.com/DataDog/dd-trace-dotnet/tree/efd756bf41b521c26eef7fc0d93b8114fd97c638) will will **decrease** line coverage by `1%`
:warning: Merging #3413 into [master](https://github.com/DataDog/dd-trace-dotnet/tree/efd756bf41b521c26eef7fc0d93b8114fd97c638) will will **decrease** branch coverage by `1%`
:no_entry: Merging #3413 into [master](https://github.com/DataDog/dd-trace-dotnet/tree/efd756bf41b521c26eef7fc0d93b8114fd97c638) will will **increase** complexity by `41`


|           | [master](https://github.com/DataDog/dd-trace-dotnet/tree/efd756bf41b521c26eef7fc0d93b8114fd97c638) | #3413       | Change   | 
|:----------|:-----------:|:-----------:|:--------:|
| Lines     | `20206` / `27205` | `19996` / `27275` |          |
| Lines %   | `74%`      | `73%`      |  `-1%` :warning:  |
| Branches  | `11808` / `16866` | `11673` / `16906` |          |
| Branches %| `70%`      | `69%`      |  `-1%` :warning:  |
| Complexity|   `18256`      | `18297`        |  `41`  :no_entry:    |

View the full report for further details:

* [HTML report for master](https://dev.azure.com/datadoghq/dd-trace-dotnet/_build/results?buildId=116979&view=codecoverage-tab) | [Source file](https://dev.azure.com/datadoghq/a51c4863-3eb4-4c5d-878a-58b41a049e4e/_apis/build/builds/116979/artifacts?artifactName=Code%20Coverage%20Report_116979&api-version=6.0&%24format=zip)  
* [HTML report for this PR #3413](https://dev.azure.com/datadoghq/dd-trace-dotnet/_build/results?buildId=117311&view=codecoverage-tab) | [Source file](https://dev.azure.com/datadoghq/a51c4863-3eb4-4c5d-878a-58b41a049e4e/_apis/build/builds/117311/artifacts?artifactName=Code%20Coverage%20Report_117311&api-version=6.0&%24format=zip)

### Datadog.Trace Breakdown :warning:

|        | [master](https://github.com/DataDog/dd-trace-dotnet/tree/efd756bf41b521c26eef7fc0d93b8114fd97c638) | #3413       | Change   | 
|:-------|:-----------:|:-----------:|:--------:|
| Lines %| `74%`      | `73%`       |  `-1%` :warning:  |
| Branches %| `70%`      | `69%`       |  `-1%` :warning:  |
| Complexity| `18256`      | `18297`       |  `41` :no_entry:  |

The following classes have significant coverage changes.

| File    | Line coverage change | Branch coverage change | Complexity change |
|:--------|:--------------------:|:----------:|:--------:|
| [Datadog.Trace.Agent.TraceSamplers.AnalyticsEventsSampler](https://github.com/DataDog/dd-trace-dotnet/tree/604ff9c1d280873f507468fb84bced756919e3f1/tracer/src/Datadog.Trace/Agent/TraceSamplers/AnalyticsEventsSampler.cs) | `-100%` :no_entry: | `-100%` :no_entry:   | `0` :heavy_check_mark: |
| [Datadog.Trace.Agent.TraceSamplers.ErrorSampler](https://github.com/DataDog/dd-trace-dotnet/tree/604ff9c1d280873f507468fb84bced756919e3f1/tracer/src/Datadog.Trace/Agent/TraceSamplers/ErrorSampler.cs) | `-100%` :no_entry: | `-100%` :no_entry:   | `0` :heavy_check_mark: |
| [Datadog.Trace.Agent.TraceSamplers.PrioritySampler](https://github.com/DataDog/dd-trace-dotnet/tree/604ff9c1d280873f507468fb84bced756919e3f1/tracer/src/Datadog.Trace/Agent/TraceSamplers/PrioritySampler.cs) | `-100%` :no_entry: | `0%` :heavy_check_mark:   | `0` :heavy_check_mark: |
| [Datadog.Trace.Sampling.GlobalSamplingRule](https://github.com/DataDog/dd-trace-dotnet/tree/604ff9c1d280873f507468fb84bced756919e3f1/tracer/src/Datadog.Trace/Sampling/GlobalSamplingRule.cs) | `-100%` :no_entry: | `0%` :heavy_check_mark:   | `0` :heavy_check_mark: |
| [Datadog.Trace.Agent.TraceSamplers.RareSampler](https://github.com/DataDog/dd-trace-dotnet/tree/604ff9c1d280873f507468fb84bced756919e3f1/tracer/src/Datadog.Trace/Agent/TraceSamplers/RareSampler.cs) | `-89%` :no_entry: | `-96%` :no_entry:   | `0` :heavy_check_mark: |
| [Datadog.Trace.Util.SamplingHelpers](https://github.com/DataDog/dd-trace-dotnet/tree/604ff9c1d280873f507468fb84bced756919e3f1/tracer/src/Datadog.Trace/Util/SamplingHelpers.cs) | `-50%` :no_entry: | `-50%` :no_entry:   | `0` :heavy_check_mark: |
| [Datadog.Trace.Telemetry.Transports.JsonTelemetryTransport](https://github.com/DataDog/dd-trace-dotnet/tree/604ff9c1d280873f507468fb84bced756919e3f1/tracer/src/Datadog.Trace/Telemetry/Transports/JsonTelemetryTransport.cs) | `-33%` :no_entry: | `-42%` :no_entry:   | `0` :heavy_check_mark: |
| [Datadog.Trace.Logging.DirectSubmission.DirectLogSubmissionManager](https://github.com/DataDog/dd-trace-dotnet/tree/604ff9c1d280873f507468fb84bced756919e3f1/tracer/src/Datadog.Trace/Logging/DirectSubmission/DirectLogSubmissionManager.cs) | `-33%` :no_entry: | `-33%` :no_entry:   | `0` :heavy_check_mark: |
| [Datadog.Trace.Logging.DirectSubmission.Sink.NullDatadogSink](https://github.com/DataDog/dd-trace-dotnet/tree/604ff9c1d280873f507468fb84bced756919e3f1/tracer/src/Datadog.Trace/Logging/DirectSubmission/Sink/NullDatadogSink.cs) | `-25%` :no_entry: | `0%` :heavy_check_mark:   | `0` :heavy_check_mark: |
| [Datadog.Trace.Processors.NormalizerTraceProcessor](https://github.com/DataDog/dd-trace-dotnet/tree/604ff9c1d280873f507468fb84bced756919e3f1/tracer/src/Datadog.Trace/Processors/NormalizerTraceProcessor.cs) | `-22%` :no_entry: | `-19%` :no_entry:   | `0` :heavy_check_mark: |
| ...And 9 more  | | | |


The following classes were added in #3413:

| File    | Line coverage | Branch coverage | Complexity |
|:--------|:--------------------:|:----------:|:--------:|
| [Datadog.Trace.RuntimeMetrics.MemoryMappedCounters](https://github.com/DataDog/dd-trace-dotnet/pull/3413/files) | `84%` | `67%` | `31` |


View the full reports for further details:

* [HTML report for master](https://dev.azure.com/datadoghq/dd-trace-dotnet/_build/results?buildId=116979&view=codecoverage-tab) | [Source file](https://dev.azure.com/datadoghq/a51c4863-3eb4-4c5d-878a-58b41a049e4e/_apis/build/builds/116979/artifacts?artifactName=Code%20Coverage%20Report_116979&api-version=6.0&%24format=zip)  
* [HTML report for this PR #3413](https://dev.azure.com/datadoghq/dd-trace-dotnet/_build/results?buildId=117311&view=codecoverage-tab) | [Source file](https://dev.azure.com/datadoghq/a51c4863-3eb4-4c5d-878a-58b41a049e4e/_apis/build/builds/117311/artifacts?artifactName=Code%20Coverage%20Report_117311&api-version=6.0&%24format=zip)

File: .azure-pipelines/ultimate-pipeline.yml (hunk 1 above)
Comment (andrewlock) [reply .azure-pipelines/ultimate-pipeline.yml:2703]: Oops, just merged... Heads up @robertpi! 🙈

**Pull request merged**